tests/utils/test_qc_check.py

```diff
@@ -0,0 +1,82 @@
+from BALSAMIC.utils.qc_check import read_hs_metrics, read_qc_table, check_qc_criteria, write_output
+from BALSAMIC.utils.qc_check import get_bait_name, get_sample_name, get_qc_criteria, failed_qc
+from BALSAMIC.utils.constants import HSMETRICS_QC_CHECK
+import os
+
+
+def test_read_hs_metrics():
+    # GIVEN the file exist
+    hs_metrics_path = "tests/test_data/qc_files/multiqc_picard_HsMetrics.json"
+
+    # WHEN reading the file
+    df = read_hs_metrics(hs_metrics_path)
+
+    # THEN check if the file contains any values
+    bol_list = df.any().tolist()
+    for n in range(len(bol_list)):
+        assert bol_list[n], "No values exists"
+
+
+def test_read_qc_table():
+    # GIVEN the file exist
+    try:
+        open("/BALSAMIC/utils/constants.py")
+    except IOError:
+        print("File doesn't exist")
+
+    # WHEN reading the file
+    df = read_qc_table(HSMETRICS_QC_CHECK)
+
+    # THEN check if the file contains any values
+    bol_list = df.any().tolist()
+    for n in range(len(bol_list)):
+        assert bol_list[n], "No values exists"
+
+
+def test_get_bait_and_sample_name():
```

Comment:
use `tumor_normal_wgs_config ` fixture here.

```suggestion
def test_get_bait_and_sample_name(tumor_normal_wgs_config):
```

Reply:
@hassanfa should tumor_normal_wgs_config be a global variable? Or how should this value be provided in the function?

Reply:
`tumor_normal_wgs_config` is a `scope="session"` fixture. It is available throughout all tests: https://docs.pytest.org/en/stable/fixture.html#conftest-py-sharing-fixture-functions Fixtures are quite handy in testing stuff, mocking, etc.

There are four fixture for configs: `tumor_normal_config`, `tumor_only_config`, `tumor_normal_wgs_config`, and `tumor_only_wgs_config`. You can use them in every tests.

Reply:
Done!